src/compiler/transformers/component-native/native-static-style.ts

```diff
@@ -65,18 +65,18 @@ const addSingleStyleGetter = (
     // static get style() { return "string"; }
```

Comment:
GH won't allow me to comment on a line higher than this one - can we add a comment before this conditional starts (above the `if` statement) that the order of the conditional must match `add-static-style.ts#getSingleStyle`?